tests/test_config.py

```diff
@@ -0,0 +1,74 @@
+import os
+import sys
+import json
+import unittest
+
+
+class TestConfigFile(unittest.TestCase):
+    """
+    This test module will run only to test manage_token() method.
+
+    Note: Numbers just after `test_` are important as they facilitate test to run in
+          specific order as `unittest` runs test in alphabetical order of thier names.
+    """
+
+
+    def test_1_nofile(self):
+        """
+        Test the module when no config.json is present in the project directory.
+        """
+
+        self.assertFalse(tweetme.manage_token('testconfig.json'))
+
+    
+    def test_2_initialfile(self):
+        """
+        Test the module when config.json is present in the project directory.
```

Comment:
The doc-strings are not required to start with `Test that` or something similar. All tests are testing something :)

Reply:
Okay let me rephrase that.